src/metadataGeneration/typeResolver.ts

```diff
@@ -432,7 +432,7 @@ export class TypeResolver {
       }) as UsableDeclaration[];
 
     if (!modelTypes.length) {
-      throw new GenerateMetadataError(`No matching model found for referenced type ${typeName}.`);
+      throw new GenerateMetadataError(`No matching model found for referenced type ${typeName}. If ${typeName} comes from a dependency, please create an interface in your own code that has the same structure. Tsoa can not utilize interfaces from external dependencies. Read more at https://github.com/lukeautry/tsoa/blob/master/ExternalInterfacesExplanation.MD`);
```

Comment:
wonder if the reference to the file should be local instead.. so `../../ExternalInterfacesExplanation.MD` or whatever. just in case the location of the repository changes at some point.

Reply:
I wanted to include the fully qualified link so that people could just copy and paste the link (or ctrl+click in vscode).

As far as the repo changing location at some point, I didn't even know that was possible.